cdap-api/src/main/java/co/cask/cdap/api/metadata/MetadataEntity.java

```diff
@@ -173,6 +183,7 @@ public String next() {
   }
 
   @Override
+  @Nonnull
```

Comment:
why? Our convention is that it can only return null if it is @Nullable. @Nonnull is redundant.

Reply:
Please respond to this comment